README.md

```diff
@@ -15,7 +17,7 @@
 - [What are GitHub Workflows?](https://jasonet.co/posts/what-are-github-workflows)
 - [Building GitHub Actions in Node.js](https://jasonet.co/posts/building-github-actions-in-node)
 - [Probot App or GitHub Action?](https://jasonet.co/posts/probot-app-or-github-action)
-<!--END_POSTS-->
+  <!--END_POSTS-->
```

Comment:
Probably unintentional? 🤷‍♂ 

```suggestion
<!--END_POSTS-->
```

Reply:
Unimportant - that's auto-generated, so if its not _broken_, I don't care 😁